Comanche055/MYSUBS.agc

```diff
@@ -89,4 +91,3 @@ DPADD2+		AD	NEGMAX		# CAN NOT OVERFLOW
 		TCF	TSK
 
 # Page 1001 (empty page)
-
```

Comment:
Apologies. Seems this change was actually valid on the previous PR - the review editor only showed this as the last line and didn't show there was another blank line below it 😢. This time i'm reviewing the entire file.